Extension/ui/settings.html

```diff
@@ -642,16 +652,20 @@
             <div class="section-title">Browse: limit symbols to included headers</div>
             <div>
             <input type="checkbox" id="limitSymbolsToIncludedHeaders" style="vertical-align: middle; transform: scale(1.5)">
-                Process only those files directly or indirectly included as headers. If not set, process all files under the specified include paths.
+                When true (or checked), the Tag Parser will only parse code files that have been directly or indirectly included by a source file 
+                in <code>${workspaceFolder}</code>. When false (or not checked), the Tag Parser will parse all code files found in the paths specified 
+                in the <code>Browse:path</code> list.

```

Comment:
Browse: path?

Reply:
Yes, should be a space to reflect the name "Browse: path" in the config UI.